app/src/main/java/org/gdg/frisbee/android/activity/AppInviteDeepLinkActivity.java

```diff
@@ -44,9 +45,7 @@
 
     @Override
     protected GoogleApiClient createGoogleApiClient() {
-        return new GoogleApiClient.Builder(this)
-            .addApi(AppInvite.API)
-            .build();
+        return GoogleApiClientFactory.createWithoutSignIn(getApplicationContext());
```

Comment:
If the client were not leaking the acitivity I am happy to use the activity.

I couldn't find different solution to resolve the leaking issue with the clients.